docs/cli/telemetry.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Telemetry
+description: Capacitor CLI Telemetry
+contributors:
+  - eric-horodyski
+---
+
+Capacitor collects anonymous telemetry data about general usage. This is an opt-out program that provides insight to the Capacitor team to help improve the product. By providing usage data, you help provide valuable insights that could shape the future of the product.
+
+You will be automatically enrolled after the Capacitor CLI successfully finishes its first command. You may opt-out at any time, wherein telemetry data will not be collected unless you opt back in. 
+
+> **Note:** Telemetry does not run in non-interactive mode. 
```

Comment:
I think is more widespread the usage of "non-interactive shells" than "non-interactive mode".

Also mention CI, like "non-interactive shells and CI server" or "non-interactive shells such as CI servers" or similar message. Not sure if all CIs are non-interactive, but the Capacitor CLI checks both things separately to not send data on any of those cases and I think specifically mentioning CI will give ease of mind to some users.